lib/widgets/content.dart

```diff
@@ -430,7 +571,13 @@ class UserMention extends StatelessWidget {
     return Container(
       decoration: _kDecoration,
       padding: const EdgeInsets.symmetric(horizontal: 0.2 * kBaseFontSize),
-      child: Text.rich(_buildInlineSpan(node.nodes, style: null)));
+      child: InlineContent(
+        // If an @-mention is inside a link, let the @-mention override it.
+        recognizer: null,  // TODO make @-mentions tappable, for info on user
+        // An @-mention can't have an embedded link, can it?  // TODO fact-check that.
```

Comment:
```dart
// An @-mention can't have an embedded link, can it?  // TODO fact-check that.
```

Wouldn't this require a user's `full_name` to contain Markdown inline-link syntax? I'm not sure if servers would parse such syntax in an @-mention, but if they do, I think we shouldn't give it an open-URL handler. An @-mention span shouldn't have a behavior where 0.0001% of the time it takes you to some website of a user's choosing, instead of opening the user-profile view in the app.